### ISSUE TITLE
Error: Cannot find module './dist/lib/generator'

### ISSUE THREAD

**Issue by boisseleau** [sirap-group/generate-swap-project#127]
Version: `v1.4.0`

**Comment by rbecheras** [comment]
Just missing the complete terminal display:

* input (the command you type)
* output (complete output: i.e. with the complete exception stack trace)

**Comment by boisseleau** [comment]
Ok i'm sorry, here is :+1: 
module.js:549
    throw err;
    ^

Error: Cannot find module './dist/lib/generator'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/boisseleau/.nvm/versions/node/v8.11.2/lib/node_modules/generate-swap-project/generator.js:1:80)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)

**Comment by rbecheras** [comment]
Created working branch (`127-error-cannot-find-module-dist-lib-generator`):

> https://github.com/sirap-group/generate-swap-project/tree/127-error-cannot-find-module-dist-lib-generator

**Comment by rbecheras** [comment]
It was a bug during the build/publish stages when releasing the `v1.4.0`.

**Comment by rbecheras** [comment]
Fixed in `v1.4.2`

Please upgrade:

```
$ npm i -g generate-swap-project@latest
```

or

```
$ npm i -g generate-swap-project@1.4.2
```
